packages/tocco-ui/src/Icon/Icon.stories.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import {storiesOf} from '@storybook/react'
+import {select, withKnobs} from '@storybook/addon-knobs'
+
+import {Icon} from './Icon'
+
+storiesOf('Icon', module)
```

Comment:
why not add knobs instead of 5 stories similar to Button

Reply:
There's more text to support each case this way.